pkg/kubernetes/difftofile.go

```diff
@@ -0,0 +1,35 @@
+package kubernetes
+
+import (
+	"github.com/pkg/errors"
+	"os"
+)
+
+// always create file for consistency
+func writeDiffToFile(path string, diff *string) error {
+	f, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+	if diff != nil {
+		_, err = f.WriteString(*diff)
+		if err != nil {
+			return err
+		}
+	}
+	return f.Close()
+}
+
+func WriteDiffToFile(path string, diff *string) error {
+	err := writeDiffToFile(path, diff)
+	if err != nil {
+		return errors.Wrap(err, "writting diff to file")
```

Comment:
Why errors.Wrap instead of fmt.Errorf?

Reply:
For consistency, they use it in Tanka. I'm not sure why.